Modules/System/Email/src/Email/Outbox/EmailEditor.Page.al

```diff
@@ -73,11 +83,21 @@ page 13 "Email Editor"
                             ToolTip = 'Specifies the email addresses of people who should receive a copy of the email.';
                             Editable = not EmailScheduled;
                             Importance = Additional;
+                            Lookup = true;
 
                             trigger OnValidate()
                             begin
                                 EmailMessage.SetRecipients(Enum::"Email Recipient Type"::Cc, CcRecipient);
                             end;
+
+                            trigger OnLookup(var Text: Text): Boolean
+                            var
+                                LookupOk: Boolean;
+                            begin
+                                LookupOk := true;
```

Comment:
you shouldn't initialize LookupOk to true. If no one subscribes and nothing happens, the lookup should return false. (same for the other 2 events)